api/src/main/java/org/openmrs/module/esaudereports/reporting/library/queries/CohortQueries.java

```diff
@@ -0,0 +1,249 @@
+package org.openmrs.module.esaudereports.reporting.library.queries;
+
+/**
+ * Classe contém as queries das cohorts de relatórios
+ * 
+ * @author Hamilton Mutaquiha
+ */
+public interface CohortQueries {
```

Comment:
We had already written this class. Is this any different from what exist? since it is NOT being implemented by any other class, we better not make it an interface